test_conformance/compiler/test_compile.cpp

```diff
@@ -16,148 +16,147 @@
 #include "testBase.h"
 #if defined(_WIN32)
 #include <time.h>
-#elif  defined(__linux__) || defined(__APPLE__)
+#elif defined(__linux__) || defined(__APPLE__)
 #include <sys/time.h>
 #include <unistd.h>
 #endif
 #include "harness/conversions.h"
 
 #define MAX_LINE_SIZE_IN_PROGRAM 1024
-#define MAX_LOG_SIZE_IN_PROGRAM  2048
+#define MAX_LOG_SIZE_IN_PROGRAM 2048
 
 const char *sample_kernel_start =
-"__kernel void sample_test(__global float *src, __global int *dst)\n"
-"{\n"
-"    float temp;\n"
-"    int  tid = get_global_id(0);\n";
+    "__kernel void sample_test(__global float *src, __global int *dst)\n"
+    "{\n"
+    "    float temp = 0.0;\n"
```

Comment:
Actually one small nitpick - this should probably be "0.0f" to avoid issues on devices that do not support fp64.

Reply:
Sure, of course.